### ISSUE TITLE
UX - Timestamp on audio player timeline hover needs to pass contrast checks for accessibility

### ISSUE THREAD

**Issue by LilyL0u** [bbc/simorgh#3665]
**Is your feature request related to a problem? Please describe.**
Currently the timestamp that appears when hovering mouse over the audio player timeline is black on red text. This is not high enough contrast to pass checks for accessibility, so the colours need to be changed e.g white on red instead.

![Image 09-09-2019 at 12 26](https://user-images.githubusercontent.com/54575620/64527142-10af6600-d2fd-11e9-8b50-94e62a406f09.jpg)

**Describe the solution you'd like**
Colour of timestamp and background should be changed to higher contrast.

**Testing notes**
Sample the colours used for the hover timestamp background and text and ensure they pass the contrast test with https://webaim.org/resources/contrastchecker/
- [x] This feature is expected to need manual testing.

**Comment by pjlee11** [comment]
This may have been fixed by bbc/morph-modules#31998 TBC

**Comment by pjlee11** [comment]
URL for testing https://www.test.bbc.com/korean/bbc_korean_radio/liveradio

**Comment by staylos92** [comment]
LGTM 👍  

Text is now white on a black background 👍
